koswat/dike_reinforcements/reinforcement_profile/berm_calculator/berm_calculator_protocol.py

```diff
@@ -0,0 +1,31 @@
+from typing import Protocol, runtime_checkable
+
+from koswat.dike.koswat_input_profile_protocol import KoswatInputProfileProtocol
+
+
+@runtime_checkable
+class BermCalculatorProtocol(Protocol):
+    """
+    Protocol for calculating the berm width, height and slope for the polderside or waterside of the dike.
+    The attributes defined here are required as they are requested by some profile calculations.
+    """
+
+    dikebase_piping_old: float
+    dikebase_piping_new: float
+    dike_height_new: float
+
+    def calculate(
+        self,
+        base_data: KoswatInputProfileProtocol,
+        reinforced_data: KoswatInputProfileProtocol,
+    ) -> tuple[float, float, float]:
```

Comment:
Return type should be self-describing object instead of a list of tuples which can represent different properties at each calculator.

Reply:
Added class `BermCalculatorResult`